### ISSUE TITLE
Is there a way to select or unselect with javascript/JQuery in a (multiple) Select element ?

### ISSUE THREAD

**Issue by NicolasPerego** [Dogfalo/materialize#6065]
Hi there,

I'm using a multiselect and I want to add a first option "select all".
The behaviour I wish is when the option is selected, all the other options are unselected.

It's very easy to do this with regular select with javascript, but I can't find a way to do this with the MD's select. I'm using the "change" event and it's perfectly detected but I can' find a way to do the selections/unselections.

In short, how, in javascript, can I select or unselect options in a multiple select ?

Thanx in advance.

**Comment by acburst** [comment]
You can make your changes on the native select and triggering the change event should propagate those changes to the material select
